packages/path_provider/path_provider_windows/example/README.md

```diff
@@ -0,0 +1,8 @@
+# path_provider_windows_example
```

Comment:
do we need a separate sample for Windows? won't the normal path_provider sample do?

Reply:
There's an open question in the longer term about how to best structure the tests and examples for federated plugins. The plugin repo tooling expects them though, and it does have value in the general case:
- It's a self-contained way to build and test the plugin, which is convenient for plugin development.
- It can test platform-specific APIs (which we don't currently have, but could in the future).

So long term, TBD, short term, this is the standard practice we've been following.